Signal/src/ViewControllers/ConversationView/Cells/OWSBubbleStrokeView.h

```diff
@@ -2,19 +2,17 @@
 //  Copyright (c) 2018 Open Whisper Systems. All rights reserved.
 //
 
+#import "OWSBubbleView.h"
+
 NS_ASSUME_NONNULL_BEGIN
 
 @class OWSBubbleView;
 
-@interface OWSBubbleStrokeView : UIView
-
-@property (nonatomic, weak) OWSBubbleView *bubbleView;
+@interface OWSBubbleStrokeView : UIView <OWSBubbleViewPartner>
```

Comment:
We now use `OWSBubbleStrokeView` to stroke "media" or "quoted replies".  "Quoted reply thumbnails" are stroked using just a layer border.